admob-lib/AdMob.cs

```diff
@@ -0,0 +1,251 @@
+using Godot;
+using System;
+
+///AUTHOR: Alexander Jungert - 28/08/2020

```

Comment:
Why did you leave that comment, do you claim ownership, this lib is MIT.

Reply:
Oh ups. Not seen. Thats from an addon within VS will remove it if shin takes it.